aiohomekit/controller/coap/connection.py

```diff
@@ -247,22 +247,12 @@ def __init__(self, owner, host, port):
         self.owner = owner
         self.pair_setup_client = None
 
-    async def reconnect_soon(self, updated_ip_port=None):
-        if updated_ip_port is None:
-            return
-        old_address = self.address
-        new_ip = updated_ip_port["AccessoryIP"]
-        new_port = updated_ip_port["AccessoryPort"]
-        new_address = f"[{new_ip}]:{new_port}"
-        logger.debug(
-            f"Device address update from zeroconf: old={old_address} new={new_address}"
-        )
-        if old_address != new_address:
-            self.address = new_address
-            await self.enc_ctx.coap_ctx.shutdown()
-            self.enc_ctx = None
-            # XXX can't .connect here w/o pairing_data
-        return
+    async def reconnect_soon(self):
+        description = self.owner.description
+        self.address = f"[{description.address}]:{description.port}"
+        await self.enc_ctx.coap_ctx.shutdown()
```

Comment:
Is this going to be an issue when hass sees it https://github.com/home-assistant/core/blob/5f827f4ca64067ca886e0e4dcb0d1498689e1ded/homeassistant/components/homekit_controller/config_flow.py#L288?

Reply:
Nope, for 2 reasons:

* The HA PR to go with this will remove that call to reconnect_soon, as we'll just use zerocnf directly instead.
* This is a different reconnect_soon. CoAPPairing.reconnect_soon used to call this reconnect_soon, but not any more. The only reconnect_soon HA can call when this merges is an empty method, so even if i forget it won't hurt.